package.json

```diff
@@ -23,8 +23,8 @@
     "webpack": "^1.12.9"
   },
   "scripts": {
-    "build": "./node_modules/webpack/bin/webpack.js",
-    "watch": "./node_modules/webpack/bin/webpack.js --watch",
-    "test": "./node_modules/mocha/bin/mocha --require ts-node/register ./test/test-*.ts"
+    "build": "node ./node_modules/webpack/bin/webpack.js",
```

Comment:
It didn't work without. (As far as i know this is a bug in npm on windows)

npm ERR! jsqr@0.2.2 build: `./node_modules/webpack/bin/webpack.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jsqr@0.2.2 build script './node_modules/webpack/bin/webpack.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jsqr package,
npm ERR! not with npm itself.